tsMuxerGUI/tsmuxerwindow.ui

```diff
@@ -2666,26 +2676,6 @@ p, li { white-space: pre-wrap; }
               </property>
              </widget>
             </item>
-            <item>
-             <widget class="QRadioButton" name="radioButtonBluRayISOUHD">
```

Comment:
You also need to remove radioButtonBluRayUHD from line 2945, otherwise you get a warning:
```
/home/test/Code/tsMuxer/tsMuxerGUI/tsmuxerwindow.ui: Warning: Tab-stop assignment: 'radioButtonBluRayUHD' is not a valid widget.
```

Reply:
Done !